### PR TITLE
Bug in error wrapping which is causing invalid wrap.

### PR DESCRIPTION
**Pull request by NoSkillGuy**

### CODE REVIEW

File: core/transaction/entity.go [hunk 1]
```diff
@@ -224,7 +224,6 @@ func VerifyTransaction(txnHash string, sharders []string) (*Transaction, error)
 			continue
 		} else {
 			if response.StatusCode != 200 {
-				customError = errors.Wrap(customError, err)
```
Comment (cnlangzi) [inline core/transaction/entity.go:227]: It is not good idea to skip error. The response error is hidden here. We don't know what happen on api request.

Comment (cnlangzi) [reply core/transaction/entity.go:227]: It is better to refactor the logic here.

**Review by cnlangzi — CHANGES_REQUESTED** [review]

File: core/transaction/entity.go (hunk 1 above)
Comment (NoSkillGuy) [reply core/transaction/entity.go:227]: Agreed, but that refactoring can go in a different PR. 

I've added this particular line in the previous commit, it wasn't there before.

**Comment by cnlangzi** [comment]
It will be fixed on https://github.com/0chain/gosdk/issues/195. let us close it first.

**Pull request closed without merging**